### DISCUSSION TITLE
SeleniumBase on Jenkins Docker containter

### DISCUSSION THREAD

**Discussion by JeffVM-Solutions** [seleniumbase/SeleniumBase#927] [Ideas]
Hello Michael!

Do you plan on creating an integration for SeleniumBase on a Jenkins Docker container? The ones for Jenkins on Azure and Google Cloud are great!

**Comment by mdmintz** [comment]
Hi @DevOps-Martins , I'm not too familiar with that particular container, but you may be able to use the SeleniumBase Dockerfile (https://github.com/seleniumbase/SeleniumBase/blob/master/Dockerfile) with Jenkins.

**Reply by JeffVM-Solutions** [reply]
I tried that but the base images are not 100% compatible.  Although Debian based, the image is not building. I'm going to keep playing around with it and see what happens. Thank you!
